zcash_client_backend/src/data_api/chain.rs

```diff
@@ -151,6 +153,7 @@ pub fn validate_chain<ParamsT, BlockSourceT>(
     parameters: &ParamsT,
     block_source: &BlockSourceT,
     validate_from: Option<(BlockHeight, BlockHash)>,
+    limit: Option<u32>,
```

Comment:
This is a change to a public API; it needs to be documented as a change in the changelog.